### PR TITLE
Create custom-sources.md

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: custom-sources.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+{% hint style="success" %}
+You can also add User Transformations to transform the payload in a specific format before sending it to the destination. Refer to the next section for more details.
+{% endhint %}
+
+## Payload Creation and Transformation
+
+This section details how RudderStack receives the data from the custom source platform and creates the resulting payload.
+
+Continuing with our Mailchimp example, let's assume that a customer subscribes to Mailchimp. Mailchimp then sends the following data (as raw content) to RudderStack:
```
Comment (ameypv-rudder) [inline custom-sources.md:62]: One query @ruchiramoitra - do all the custom source platforms send the data in a raw content format?

File: custom-sources.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+![](https://user-images.githubusercontent.com/59817155/127986131-3740dbfe-7d55-4328-abe1-63fb8ac70de2.png)
+
+{% hint style="info" %}
+Remember to add and test your webhook URL.
+{% endhint %}
+
+* When the users perform any of the actions associated with the source, the source platform will send the generated events to the webhook URL. 
+
+In this example, Mailchimp sends the updates under **What type of updates should we send?** (seen in the image above) as user events to the webhook URL with the content type `application/x-www-form-urlencoded`.
```
Comment (ameypv-rudder) [inline custom-sources.md:46]: Is the content type `application/x-www-form-urlencoded` applicable just for Mailchimp or for all custom sources?

Comment (ruchiramoitra) [reply custom-sources.md:46]: This is a particular case for mail chimp. It can be for others too...

File: custom-sources.md (hunk 1 above)
Comment (ruchiramoitra) [reply custom-sources.md:62]: No I have written it as a raw content below. It will not in most cases as raw content

**Review by arnab-p — APPROVED** [review]

**Pull request merged**